Tests/Encrption/EncryptionTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zest\Encryption\Adapter\OpenSslEncryption;
+use Zest\Encryption\Adapter\SodiumEncryption;
+
+class EncryptionTest extends TestCase
+{
+    public function testOpenSslEncrypt()
+    {
+        $openSslEncryption = new OpenSslEncryption('123456key');
+        $str = 'This is a string';
+        $encryptHash = $openSslEncryption->encrypt($str);
+        $this->assertNotEquals($str, $encryptHash);
+        $this->assertEquals($str, $openSslEncryption->decrypt($encryptHash));
+    }
+
+    public function testSodiumEncrypt()
+    {
+        $sodiumEncryption = new SodiumEncryption('123456key');
+        $str = 'This is a string';
+        $encryptHash = $sodiumEncryption->encrypt($str);
+        $this->assertNotEquals($str, $encryptHash);
+        $this->assertEquals($str, $sodiumEncryption->decrypt($encryptHash));
```

Comment:
Do you know the `assertEquals` and `assertSame`?

The `assertEquals` is about letting expected value be same as result value.

It's same as the `==` operator.

And their types will be different possibly.

The `assertSame` is about letting expected value be same as result value.

It's same as the `===` operator.

If we want to assert the expected value is equal to result, using the `assertEquals` is fine I think.

If we want to assert the expected value and type are same as result, using the `assertSame` should be proper I tihnk.